### ISSUE TITLE
Unused in y_true in loss function

### ISSUE THREAD

**Issue by bdradteam** [jaredleekatzman/DeepSurv#41]
Generally, I believe for a neural network, the loss function consists of comparing the output from the neural network (y_pred) with the already assigned labels/data (y_true). I realize for your lost function, you dod not use y_true, which is the known time events, how can you get the loss updated this way?
